themes/vocabulary_theme/templates/blocks/hero.html

```diff
@@ -12,6 +12,6 @@ <h1>
     </div>
   </div>
   <div class="level-right hero-image">
-    <img class="image" src="./github.svg" />
+    <img class="image" src="./github.svg"/>
```

Comment:
please re-add this space you've removed (and on any other `img` tags where this has occurred.